server/plugins/connect-plugin-api/src/main/java/com/oceanbase/odc/plugin/connect/api/InformationExtensionPoint.java

```diff
@@ -27,4 +29,8 @@
 public interface InformationExtensionPoint extends ExtensionPoint {
 
     String getDBVersion(Connection connection);
+
+    String getDBVersionComment(Connection connection);
```

Comment:
Oracle and other db may not provide version comment, this method can move to OBMySQLInformationExtensionPoint for present